challenge_serving_drinks/scrips/challenge_serving_drinks

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python
```

Comment:
Please be consistent with the shebang lines. Please use `#!/usr/bin/env python`

Reply:
Updated in df09e6cc9bdb5fa7ee45cfd35e6ff2b842b08f56